xbmc/utils/EGLUtils.cpp

```diff
@@ -669,6 +669,12 @@ void CEGLContextUtils::SetDamagedRegions(const CDirtyRegionList& dirtyRegions)
   {
     EGLint height = 1080;
     eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);
```

Comment:
Is it possible to use
```
if (eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height) != EGL_TRUE)
```
or
```
if (!eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height))
```
to prevent calling `eglGetError()`?